_translator-files/po/ko_KR/FAQ.po

```diff
@@ -92,142 +90,139 @@ msgstr "**Jamulus를 사용할 때 발생하는 일반적인 문제와 해결 
 #: ../wiki/en/FAQ.md:28
 msgid "Bear in mind also that all participants should follow [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) which will also determine whether you can play in time properly."
-msgstr ""
+msgstr "또한 모든 참가자는 [황금률](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)을 따라야 하며 시간에 맞춰 제대로 플레이할 수 있는지 여부도 결정됩니다."
 
 #. type: Title ###
 #: ../wiki/en/FAQ.md:29
 #, no-wrap
 msgid "How do I know if I can join a Server? Are there rules?"
-msgstr ""
+msgstr "서버에 참여할 수 있는지 어떻게 알 수 있나요? 규칙이 있나요?"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:32
 msgid "If somebody lists a Server on one of the Directories built into Jamulus, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. Whilst some Servers may state their policies in the welcome message you will see in the chat window, Jamulus itself does nothing to enforce these."
-msgstr ""
+msgstr "누군가 Jamulus에 내장된 디렉터리 중 하나에 서버를 나열하면 누구나 해당 서버에서 플레이할 수 있다는 것을 인정합니다. Jamulus에는 비밀번호 보호나 기타 인증 메커니즘이 없습니다. 일부 서버는 채팅 창에 표시되는 환영 메시지에 정책을 명시할 수 있지만 Jamulus 자체는 이를 시행하기 위해 아무것도 하지 않습니다."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:34
 msgid "Note also that Servers do not have to be registered on a Directory in order for Jamulus Clients to connect to them. Server operators can just give out the address of their Servers to those they want to play with, and they can then connect directly. Have a look at [this overview for more information](/wiki/Running-a-Server#server-modes)."
-msgstr ""
+msgstr "또한 Jamulus 클라이언트가 서버에 연결하기 위해 서버를 디렉터리에 등록할 필요는 없습니다. 서버 운영자는 플레이하려는 서버의 주소를 제공하고 직접 연결할 수 있습니다. [자세한 내용은 이 개요](/wiki/Running-a-Server#server-modes)를 참조하세요."
 
 #. type: Title ###
 #: ../wiki/en/FAQ.md:35
 #, no-wrap
 msgid "Why shouldn’t I use wireless equipment?"
-msgstr ""
+msgstr "무선 장비를 사용하면 안 되는 이유는 무엇인가요?"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:38
 msgid "Whether or not you will be able to play in time with other musicians depends mainly on how much latency (delay) you have in your sound signal and whether you are all following [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together). An overall delay much bigger than 50 ms usually makes it too hard to play in time unless you train yourself to do so. Some musicians report they can play in time with delays of up to 70 ms by playing ahead of the beat. But much beyond that and it becomes too difficult unless timing isn’t an important factor in the music or you just want to listen."
-msgstr ""
+msgstr "다른 뮤지션들과 함께 제 시간에 연주할 수 있는지 여부는 주로 사운드 신호의 레이턴시(딜레이) 얼마나 되는지, 그리고 모두가 [황금률](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)을 따르고 있는지 여부에 따라 달라집니다. 50ms보다 훨씬 큰 전체 딜레이는 일반적으로 훈련하지 않는 한 시간에 연주하는 것을 너무 어렵게 만듭니다. 일부 뮤지션들은 비트보다 앞서 연주함으로써 최대 70ms의 딜레이에 맞춰 연주할 수 있다고 보고합니다. 하지만 그 이상이면 타이밍이 음악에서 중요한 요소가 아니거나 단순히 듣기만 원하는 경우가 아니라면 너무 어려워집니다."
```

Comment:
Just as quick note - the links will not be broken, but the anchor just fail. I'm unsure how the actual ID would look like for non ASCII headlines.